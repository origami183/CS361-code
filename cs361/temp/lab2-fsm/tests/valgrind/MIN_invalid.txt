==1359228== Memcheck, a memory error detector
==1359228== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==1359228== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==1359228== Command: ../fsm 0 2 1 2 3 3 5 6
==1359228== 
NEW.ADMIT -> RDY
RDY.SCHEDULE -> NST
RDY.DISPATCH -> RUN
RUN.SCHEDULE -> RDY
RDY.BLOCK -> BLK
BLK.BLOCK -> NST
BLK.EXIT -> NST
BLK.KILL -> TRM
==1359228== 
==1359228== HEAP SUMMARY:
==1359228==     in use at exit: 0 bytes in 0 blocks
==1359228==   total heap usage: 2 allocs, 2 frees, 8,224 bytes allocated
==1359228== 
==1359228== All heap blocks were freed -- no leaks are possible
==1359228== 
==1359228== For lists of detected and suppressed errors, rerun with: -s
==1359228== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
