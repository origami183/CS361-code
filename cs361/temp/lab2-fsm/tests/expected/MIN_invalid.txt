NEW.ADMIT -> RDY
  [Process is live]
RDY.SCHEDULE -> NST
RDY.DISPATCH -> RUN
RUN.SCHEDULE -> RDY
RDY.BLOCK -> BLK
  [Process is blocked]
BLK.BLOCK -> NST
BLK.EXIT -> NST
BLK.KILL -> TRM

  Process stats:
    runtime: 1
    normal exit: false
